README.md

```diff
@@ -17,7 +17,7 @@ Datapath plugins are named by URI schemes. Internally we have the following low-
 
 ## Build & installation
 
-Install dependencies: `cmake3`, `make`, `python-setuptools`, `nbd`, `python-psutil`, `qemu-dp`, `systemd` and `xapi-storage`.
+Install dependencies: `cmake3`, `make`, `python-setuptools`, `nbd`, `qemu-dp`, `systemd` and `xapi-storage`.
```

Comment:
qemu-dp is in the same bucket, but I wouldn't spend too much time on the readme for a codebase that we're highly likely to scrap anyway - not mentioning that if we happen to resurrect qemu-dp on this codebase we risk to forget adding them back